integrations/segment/src/events.ts

```diff
@@ -6,12 +6,12 @@ import { version } from '../package.json';
  * Generate the event to track in Segment for an actual GitBook event.
  */
 export function generateSegmentTrackEvent(event: api.SiteViewEvent) {
-    const { visitor, referrer, url, spaceId, pageId, siteId } = event;
+    const { visitor, referrer, url, siteId } = event;
 
     const anonymousId = getAnonymousId(event);
     const visitedURL = new URL(url);
     return {
-        event: '[GitBook] space_view',
+        event: '[GitBook] site_view',
```

Comment:
Why do we need `[GitBook]`?

Reply:
It will show up in the segment side of things - might be nice to have in case users have multiple sources so they know which ones are coming from GitBook